hooli_data_eng/assets/forecasting/__init__.py

```diff
@@ -108,33 +137,83 @@ def predicted_orders(
     predicted_data = model_func(x=future_dates.astype(np.int64), a=a, b=b)
     return pd.DataFrame({"order_date": future_dates, "num_orders": predicted_data})
```

Comment:
I received an error when I tried to run this asset, which blocks me from testing the databricks_asset:

`dagster._core.errors.DagsterInvariantViolationError: Snowflake I/O manager: The Snowflake column ORDER_DATE in table predicted_orders is of type VARCHAR(16777216) and should be of type TIMESTAMP to store the time data in dataframe column ORDER_DATE. Please migrate this column to be of time TIMESTAMP_NTZ(9) to store time data.`